Source/NaturalLanguageClassifierV1/Tests/NaturalLanguageClassifierTests.swift

```diff
@@ -0,0 +1,320 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import XCTest
+import WatsonDeveloperCloud
+
+class NaturalLanguageClassifierTests: XCTestCase {
```

Comment:
We pulled an interesting trick in the `master` branch's tests. We had a few goals:

- We wanted to use a trained classifier to check the `classify` endpoint.
- Classifiers take time to be trained, but we didn't want to wait for training to complete every time we ran the tests.
- We wanted anyone with credentials for the Natural Language Classifier service to be able to run the tests.

To satisfy those goals, we needed a trained classifier when running the tests. But not everyone would already have a classifier trained with our training data. So we wrote [this function](https://github.com/watson-developer-cloud/ios-sdk/blob/master/WatsonDeveloperCloud/NaturalLanguageClassifier/Tests/NaturalLanguageClassifierTests.swift#L163) that checks for a trained classifier and creates one if it doesn't already exist.

I think your tests look very solid right now, but you might want to add a similar function so that people outside of our team can have a classifier created to be used for training.

On that note, since we want the classifier to already be trained before running the tests, we don't want to delete it after executing the tests (i.e. I don't think you need the `deleteStaleClassifiers()` method).